cfgov/unprocessed/js/modules/util/ajax-request.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+/**
+* Initiates an ajax request when passed a type, url,
+* and an optional opts object.
+*
+* @param {string} type request type (GET, POST)
+* @param {string} url request url
+* @param {Object} opts object containing optional
+* request parameters, including data, headers,
+* and success, fail, and done callbacks
+* @returns {Object} xhr
+*/
+
+function ajaxRequest( type, url, opts ) {
```

Comment:
Yes, I was thinking this could also be used for `FormSubmit`. I'll make that change in a separate PR, though, since it touches so much of the site.